src/sd_api_v3/sdk/components/serialization/common/ser_config.h

```diff
@@ -39,8 +39,8 @@ extern "C" {
 
 /** Max packets size in serialization HAL Transport layer (packets before adding PHY header i.e.
  *  packet length). */
-#define SER_HAL_TRANSPORT_APP_TO_CONN_MAX_PKT_SIZE    (uint32_t)(384)
-#define SER_HAL_TRANSPORT_CONN_TO_APP_MAX_PKT_SIZE    (uint32_t)(384)
+#define SER_HAL_TRANSPORT_APP_TO_CONN_MAX_PKT_SIZE    (uint32_t)(500)
+#define SER_HAL_TRANSPORT_CONN_TO_APP_MAX_PKT_SIZE    (uint32_t)(500)
```

Comment:
This change is also done by sdk122_connectivity.patch. Do we need it here as well?

Reply:
We need it here as well since this applies to the app side pc-ble-driver project (x86), whereas the sdk122_connectivity.patch only applies to the connectivity Keil ARM project.